packages/playwright-test/src/third_party/tsconfig-loader.ts

```diff
@@ -188,6 +183,17 @@ function loadTsconfig(
   return config;
 }
 
+function mergeConfigs(base: Tsconfig, override: Tsconfig): Tsconfig {
+  return {
+    ...base,
```

Comment:
IIUC, the `references` property should be ignored: https://www.typescriptlang.org/tsconfig#extends